### ISSUE TITLE
Icon not showing in Zeal

### ISSUE THREAD

**Issue by megies** [zealdocs/zeal#264]
The Dash standard docs at http://kapeli.com/docsets#addingicon state that..
```
To set a custom icon for your docset,
simply add a "icon.png" file directly inside the docset bundle.
```
..but Zeal is not reading that file but as stated [here](https://github.com/zealdocs/zeal/issues/186#issuecomment-56822828) is rather looking for a file `Contents/Resources/Documents/icon.png`. For that reason docsets created using `doc2dash` are not displayed with their icon in Zeal.  (see hynek/doc2dash#35)

**Comment by megies** [comment]
Quick squashing this issue! Another thing I noticed is that some docsets I've got from a Dash user have the file `Info.plist` with a lower case first letter "i": `info.plist`. These turn up with an empty docset name in the list of docsets in the left pane. That should also be a quick fix.. (I guess here: https://github.com/zealdocs/zeal/blob/8532a6262b37fccf280af03dfa84dd277d42a94a/src/registry/docset.cpp#L27)
